Source/AwsCommonRuntimeKit/io/TlsConnectionOptions.swift

```diff
@@ -5,35 +5,35 @@ import AwsCIo
 
 public final class TlsConnectionOptions {
 	private let allocator: Allocator
-    var rawValue: UnsafeMutablePointer<aws_tls_connection_options>
-
+    private var _rawValue: UnsafeMutablePointer<aws_tls_connection_options>
```

Comment:
I thought it was weird that you didn't convert this one to CStruct

But looking at it, it is a bit different. This options-struct has init/cleanup functions and setters, instead of being a plain old non-owning struct.

We **could** still use the CStruct protocol though. its `withCStruct()` function would call init(), then the setters, then invoke the body callback, and then call clean_up() afterwards

The setters can't ACTUALLY fail if you look into them. They can only fail if allocation fails, but we don't let allocation fail anymore, so we can skip checking those errors

OR

keep TlsConnectionOptions a class, but have it *feel* like the other options-structs by using swift properties, instead of setter functions. And we could still have a `withCStruct()` function that just delivered the pre-existing struct

Reply:
Thanks, fixed. There is one other class TlsContextOptions which has a similar issue. Looking into if we can make it a struct instead.